markdown/GameGuides/DeadSpace.md

```diff
@@ -12,15 +12,42 @@ Reshade | Yes
 Ansel | No
 DirectX versions | 9
 
-## Tools
+Resolution can be changed in configuration file.
+GeDoSaTo is supported.
+[(guide)](https://steamcommunity.com/sharedfiles/filedetails/?id=604010024)  
+
+## Free Camera
+[Table by GhostInTheCamera](https://github.com/ghostinthecamera/PhotomodeCheatTables/blob/master/WIP/Dead%20Space/GITC%20-%20Dead%20Space%20WIP.CT)  #
+[(download)](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/ghostinthecamera/PhotomodeCheatTables/blob/master/WIP/Dead%20Space/GITC%20-%20Dead%20Space%20WIP.CT)
+
+### Description:  
+Turn Free Camera on/off with `Insert` and `Delete`:
+* for camera navigation/rotation - use `Num 1-9` and arrows.
+* change FOV with `Num +/-`.
+* char can be moved in that mode. 
+### Other features:  
+* Timeflow: turn  off/on with `Num 0/.` - used to pause (freeze) the game. 
+* Modify Timescale and Aspect Ratio.    
+
+### Hotkeys Modification
+[Cheat Table](https://gist.github.com/r3sus/93edc2c58a25841c30b6effcf7936257)  #
```

Comment:
Please include tables in the PR so they're in the repository. Don't link to gists which can be deleted at any moment.